lbp/src/components/TokenSale/TokenSaleInfo.tsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { Col, Row } from "react-bootstrap";
+import styled from "styled-components";
+
+import {
+  BaseLink,
+  PrimaryText,
+  SecondaryText,
+  Title,
+} from "shared/lib/designSystem";
+import colors from "shared/lib/designSystem/colors";
+import theme from "shared/lib/designSystem/theme";
+import {
+  getEtherscanURI,
+  RibbonTokenAddress,
+} from "shared/lib/constants/constants";
+import { ExternalIcon } from "shared/lib/assets/icons/icons";
+
+const TokenSaleRow = styled(Row)`
+  margin-top: 64px;
+`;
+
+const InfoTitle = styled(Title)`
+  font-size: 18px;
+  line-height: 20px;
+`;
+
+const InfoText = styled(PrimaryText)`
+  color: ${colors.text};
+  font-weight: 400;
+`;
+
+const InfoTable = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  border: ${theme.border.width} ${theme.border.style} ${colors.border};
+  border-radius: ${theme.border.radiusSmall};
+  background: ${colors.primaryText}03;
+  margin: 40px 0px;
+`;
+
+const InfoTableCell = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 50%;
+  padding: 16px;
+
+  &:not(:nth-child(-n + 2)) {
+    border-top: ${theme.border.width} ${theme.border.style} ${colors.border};
+  }
+
+  &:nth-child(2n + 1) {
+    border-right: ${theme.border.width} ${theme.border.style} ${colors.border};
+  }
+`;
+
+const LabelText = styled(SecondaryText)`
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: normal;
+  margin-bottom: 4px;
+`;
+
+const InfoTableCellData = styled(Title)`
+  letter-spacing: 1px;
+`;
+
+const TokenAddress = styled(Title)`
+  color: ${colors.text};
+`;
+
+const TokenSaleInfo = () => {
+  return (
+    <TokenSaleRow>
+      <Col md={6} className="d-flex flex-column">
+        {/* Text wall */}
+        <InfoTitle>RBN TOKEN SALE</InfoTitle>
+        <InfoText className="mt-3">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit ut aliquam,
+          purus sit amet luctus venenatis, lectus magna fringilla urna,
+          porttitor rhoncus dolor purus non enim praesent elementum facilisis
+          leo, vel fringilla est ullamcorper eget nulla facilisi etiam dignissim
+          diam quis enim lobortis scelerisque fermentum dui faucibus in ornare
+          quam viverra orci sagittis eu volutpat odio facilisis mauris sit amet
+          massa vitae tortor condimentum lacinia quis vel eros donec ac odio
+          tempor orci dapibus ultrices in iaculis nunc sed augue lacus
+        </InfoText>
+
+        {/* Table */}
+        <InfoTable>
+          <InfoTableCell>
+            <LabelText>$RBN Start Balance</LabelText>
+            <InfoTableCellData>30,000,000</InfoTableCellData>
+          </InfoTableCell>
+          <InfoTableCell>
+            <LabelText>USDC Start Balance</LabelText>
+            <InfoTableCellData>5,000,000</InfoTableCellData>
+          </InfoTableCell>
+          <InfoTableCell>
+            <LabelText>$RBN Start Weight</LabelText>
+            <InfoTableCellData>95</InfoTableCellData>
```

Comment:
Need to confirm copy here, but should be `95%`. Because the 95 could refer to actual weights that's on the contract

Reply:
Let's confirm with Vinnie